.storybook/package.json

```diff
@@ -42,7 +42,7 @@
 		"@adobe/spectrum-css-workflow-icons": "^1.5.4",
 		"@spectrum-css/bundle": "1.0.0",
 		"@spectrum-css/tokens": "16.0.1",
-		"@spectrum-css/tokens-legacy": "npm:@spectrum-css/tokens@^15.2.0",
+		"@spectrum-css/tokens-legacy": "npm:@spectrum-css/tokens@^15.0.0",
```

Comment:
There seems to be an issue with the 15.2.0 tokens.json file so I pulled back to 15.0.0  where no issues exist.